src/ToeplitzMatrices.jl

```diff
@@ -108,9 +113,11 @@ function mul!(C::StridedMatrix{T}, A::AbstractToeplitz{T}, B::StridedMatrix, α:
     return C
 end
 # Avoid ambiguity error
+#=
```

Comment:
Just delete code if it's not needed unless you expect it to be needed in the future: git history will save it around in case

Reply:
Done.